tests/test_algorithm.py

```diff
@@ -4755,10 +4366,11 @@ def make_equity_info(cls):
             orient='index',
         )
 
-    @classmethod
-    def init_class_fixtures(cls):
-        super(TestOrderAfterDelist, cls).init_class_fixtures()
-        cls.data_portal = FakeDataPortal(cls.env)
+    # XXX: This suite doesn't use the data in its DataPortal; it uses a
+    # FakeDataPortal with different mock data.
+    def init_instance_fixtures(self):
```

Comment:
This needs to be in `init_instance_fixtures` because `WithDataPortal` sets `self.data_portal` as a instance attribute, which takes precedence over a class-level attribute.